pager/src/main/java/com/google/accompanist/pager/Pager.kt

```diff
@@ -465,9 +468,15 @@ private class ConsumeFlingNestedScrollConnection(
     }
 
     override suspend fun onPostFling(consumed: Velocity, available: Velocity): Velocity {
-        // We can consume all post fling velocity on the main-axis
-        // so that it doesn't propagate up to the Pager
-        return available.consume(consumeHorizontal, consumeVertical)
+        return if (pagerState.currentPageLayoutInfo?.offset != 0) {
```

Comment:
Hi, do you think we could use currentPageOffset here? It's already public and if currentPageLayoutInfo.offset == 0, then this will also be 0.

Reply:
Ah, you are right. That's cool!
I will commit it soon.